bin/checkPad.js

```diff
@@ -5,29 +5,26 @@
 
 // As of v14, Node.js does not exit when there is an unhandled Promise rejection. Convert an
 // unhandled rejection into an uncaught exception, which does cause Node.js to exit.
-process.on('unhandledRejection', (err) => { throw err; });
 
-const npm = require('ep_etherpad-lite/node_modules/npm');
-const util = require('util');
+const process = require('process');
```

Comment:
You don't need to `require('process')`—the `process` object is a global variable.